modules/internal/LocaleModelHelpSetup.chpl

```diff
@@ -35,6 +35,11 @@ module LocaleModelHelpSetup {
 
   config param debugLocaleModel = false;
 
+  // We would really like a class-static storage class.(C++ nomenclature)
```

Comment:
I can probably delete that variable, because it is never set to anything but false.  I think I will leave that hook there in case it is needed later, though, and just delete the comment.  It works just as well as a static class member would, so the comment is not needed.